cfgov/v1/models/snippets.py

```diff
@@ -9,19 +9,31 @@
 
 @register_snippet
 class Contact(models.Model):
-    heading = models.CharField(max_length=22, blank=False)
+    title = models.CharField(verbose_name=('Title'), max_length=255,
+                             help_text=("The snippet title as you'd like it to be seen by the public"))
+    slug = models.SlugField(verbose_name=('Slug'), max_length=255,
```

Comment:
So a couple things.
1. Snippets are not in the page tree. The are just database tables that store data for reuse across the site. 
2. If we're going this route, I'm going to suggest that we use a BaseSnippet class.
3. Identifying a slug that needs to be entered in is onerous and requires a weird clean() method not implemented.

I can create the BaseSnippet class, that does this checking. Stand by for PR.